app/blueprints/fund_builder/routes.py

```diff
@@ -204,80 +208,208 @@ def clone_round(round_id, fund_id):
 
 
 @build_fund_bp.route("/fund", methods=["GET", "POST"])
-def fund():
+@build_fund_bp.route("/fund/<fund_id>", methods=["GET", "POST"])
+def fund(fund_id=None):
```

Comment:
Can we add some logs based on criticality ?

Reply:
Good shout, added logs for add, update fund and round.